### DISCUSSION TITLE
Starbases

### DISCUSSION THREAD

**Discussion by DominusExMachina** [bcholmes/StarTrek2d20#140] [Ideas]
While we're all grateful that we have a starship creator (especially after the Utopia Planitia sourcebook came out), it seems that starbases - which were also featured in said sourcebook - were conspicuously left out. Was this a conscious choice, or are there plans to include a starbase creator in future releases?

**Comment by bcholmes** [comment]
There's an issue (#82) in the backlog. Just haven't gotten around to it.

**Reply by matthew81** [reply]
Has there been any progress on this yet?

**Comment by KeptinJurk** [comment]
seconding this! we would love and worship the capacity to build star bases.

i am personally wanting to build a mobile space station, using the UP book's rules, giving it real engines and warp drive, to make a mobile base for fleet repair and refit. thataway I can move the base and its FLEET of ships docked inside it, TO a brand new unexplored sector/s and have them and their crews operate out of the station, sorta like narendra station and the shackleton expanse campaign, except on crack.

**Comment by ww123ist** [comment]
I concur! I have a huge campaign starting up in a few months that's starbase-based, and we'd love to have the official page from your wonderful generator for combat. I know you said starbase building was one of your main goals for this year, but know that your community is really excited for that to be top of the list!

**Comment by KeptinJurk** [comment]
TA DA! look how COOL you can build shit for STA with the UP book. this baby can warp to entire brand spanking new sectors and disgorge 6 scale 6 ships. that's a fleet of galaxies. but because I am in lust and love my my beloved EXCELSIOR!!!, THIS particular space station has a fleet of excelsior's assigned to her permanently, of course.

![DS13 A](https://github.com/user-attachments/assets/e768608a-60f0-4385-a52e-f21919c2311e)
![DS13 B](https://github.com/user-attachments/assets/82fcccfd-d8f4-4862-9c32-48c095db0575)
![DS13 C](https://github.com/user-attachments/assets/9b0b9915-783d-468b-8332-d8db39ba5f2b)
